## FootStatePacketMessage
# This message tells the controller whether a foot is load bearing.

#Options for enum
# uint8 L = 0
# uint8 R = 1
uint8 robotSide
bool loadBearing

